src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+import Icon from "../Icon";
+import PaginationButton from "./PaginationButton/PaginationButton";
+import { PaginationProps } from "./types";
+
+const Pagination: FC<PaginationProps> = ({
+  children,
+  onLoadNext,
+  onLoadPrevious,
+  pageNum,
+  pagesAmount
+}) => {
+  return (
+    <>
+      <PaginationButton
+        intent={"previous"}
+        icon={<Icon iconName={"arrowUp"} width="16" height="16" />}
+        onClick={onLoadPrevious}
+        isVisible={pageNum !== 0}
+      />
+      {children && children}
+      <PaginationButton
+        icon={<Icon iconName="arrowDown" width="16" height="16" />}
```

Comment:
We have two different approaches to setting iconName here - one with `{}` and here without them - please unify